ert3/console/__init__.py

```diff
@@ -66,6 +67,23 @@ def _run_experiment(workspace_root, experiment_name):
     if _experiment_have_run(experiment_root):
         raise ValueError(f"Experiment {experiment_name} have been carried out.")
 
+    coefficients = [
```

Comment:
Feel free to move the sampling into a separate, private function..

Reply:
We went a bit too far I believe, as we've already started on the next issue.. But I don't mind refactoring a bit and still include this part.